README.md

```diff
@@ -49,15 +49,21 @@ npm install -g gitmoji-changelog@alpha
 
 * [yvonnick/gitmoji-changelog:latest](https://github.com/frinyvonnick/gitmoji-changelog/blob/master/Dockerfile)
 
-## Install
 
+## 🚀 Usage (with npx)
+
+Make sure you have [npx](https://www.npmjs.com/package/npx) installed (`npx` is shipped by default since npm `5.2.0`)
```

Comment:
Didn't know about npx being shipped by default since npm 5.2. Nice thing 👍

Reply:
Yes ! I've seen this nuance on [nodejs.dev](https://nodejs.dev/the-npx-nodejs-package-runner) 😄